frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -234,10 +211,7 @@ const composeAndPushNotificationMessages = (
     >
         <div class="page-body">
             <form id="grantAccessForm" class="form-container">
-                <StepContainer
-                    title="User information"
-                    :subtitle="`Enter the user information to add a new user to ${selectedApplicationDisplayText}`"
-                >
```

Comment:
The removal of the `${selectedApplicationDisplayText}` might make this "GrantAccess" page confusing for the admin as they will not know easily which "application" they are assigning users for.
Might need to discuss with @OlgaLiber2 what's the better text to indicate the "application" was selected for granting.
@MCatherine1994

Reply:
I agree with @ianliuwk1019  on this matter, maybe we need to come up with a better text for the subtitle item, not removing it altogether. My suggestion is to revert it to what it was while we discuss it. @MCatherine1994 @OlgaLiber2